packages/form-core/src/choice-group/ChoiceGroupMixin.js

```diff
@@ -207,11 +207,34 @@ const ChoiceGroupMixinImplementation = superclass =>
     }
 
     /**
-     * @override
+     * A filter function which will exclude a form field when returning false
+     * By default, exclude form fields which are disabled
+     *
+     * The type is be passed as well for more fine grained control, e.g.
+     * distinguish the filter when fetching modelValue versus serializedValue
+     *
+     * @param {FormControl} el
+     * @param {string} type
+     * @returns {boolean}
+     */
+    // eslint-disable-next-line class-methods-use-this, no-unused-vars
+    _getFromAllFormElementsFilter(el, type) {
+      return true;
+    }
+
+    /**
+     * Implicit :( @override for FormGroupMixin, as choice fields "fieldsets"
+     * will always implement both mixins
+     *
+     * TODO: Consider making this explicit by extracting this method to its own mixin and
+     * using it in both FormGroupMixin and ChoiceGroupMixin, then override it here
+     * This also makes it more DRY as we have same method with similar implementation
+     * in FormGroupMixin. I (@jorenbroekema) think the abstraction is worth it here..
+     *
      * @param {string} property
      * @protected
      */
-    _getFromAllFormElements(property, filterCondition = () => true) {
+    _getFromAllFormElements(property) {
```

Comment:
Maybe `_getFromAllFormElements(property, filterCondition = this. _getFromAllFormElementsFilter) {` to keep it backw. compatible?

Reply:
Not sure how it would make it backward compatible? Compatible with what exactly?

Also, are we concerned with backwards compatibility for lion, which is v0? As far as I know this should not be problematic for the ING lion extension

Reply:
Basically with all Subclassers that use this protected api (calling this method directly, regardless of what is configured on prototype level(could still be a use case)). Problem is that we don't have a good overview of all Subclassers and what they do. 
But I think breaking changes should be avoided as much as possible, unless unavoidable: lit major version, shadow parts (even these could be implemented in a backwards compatible way).

Reply:
Hmm I guess it could be possible that a subclasser relies on this method and relies on that default parameter "filterCondition"

I definitely disagree that we should avoid breaking changes as minor as this one. Lion should be kept clean, no unnecessary backwards compatibility please. This is a breaking change that can be easily alleviated on the subclasser's part, we just need to release it as a minor and mark it as breaking in the changelog, and the subclasser can deal with it.

Reply:
I'm also for keeping it clean... if an extension layer really needs it they can override it in that extension layer